ast/misc.go

```diff
@@ -228,7 +277,20 @@ type ExecuteStmt struct {
 
 // Restore implements Node interface.
 func (n *ExecuteStmt) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	ctx.WriteKeyWord("EXECUTE ")
+	ctx.WriteName(n.Name)
+	if n.UsingVars != nil && len(n.UsingVars) > 0 {
+		ctx.WriteKeyWord(" USING ")
+		for i, val := range n.UsingVars {
+			if i != 0 {
+				ctx.WritePlain(",")
+			}
+			if err := val.Restore(ctx); err != nil {
+				return errors.Annotate(err, fmt.Sprintf("An error occurred while restore ExecuteStmt.UsingVars index %d", i))
```

Comment:
```suggestion
				return errors.Annotatef(err, "An error occurred while restore ExecuteStmt.UsingVars index %d", i)
```

Reply:
thank you for suggestion，it has been fixed！